Automattic/wp-calypso
# Add light color scheme from wp-admin

One part of #47036

#### Changes proposed in this Pull Request

* Add light color scheme

#### Testing instructions

* Visit /me/account
* Choose light and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:
![2020-11-06_14-59](https://user-images.githubusercontent.com/937354/98414302-badd6780-2040-11eb-9aaf-f4a573d0ca73.png)

Wp-Admin
![2020-11-06_15-00](https://user-images.githubusercontent.com/937354/98414321-c892ed00-2040-11eb-87e7-2dc7f6dfd804.png)


#### Notes

This seems to be quite an odd scheme - almost an inverted scheme when it comes to the sidebar.
